app/Console/Command/ChangePlaceTypeOnPlacesWithEventEventType.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemIdentifier;
+use CultuurNet\UDB3\Offer\Commands\UpdateType;
+use CultuurNet\UDB3\Place\Commands\Moderation\Reject;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\StringLiteral;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class ChangePlaceTypeOnPlacesWithEventEventType extends AbstractCommand
+{
+    private SearchServiceInterface $searchService;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService
+    ) {
+        $this->searchService = $searchService;
+        parent::__construct($commandBus);
+    }
+
+    public function configure(): void
+    {
+        $this->setName('place:eventtype:update:reject');
+
+        $this->setDescription(
+            'Updates all places with an Event eventType and optionally reject them'
+        );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        $logger = new ConsoleLogger($output);
+
+        $eventEventType = $this->askEventEventType($input, $output);
```

Comment:
If I read the ticket it seems there is a mapping between the existing event type and the new event type. But with this script both the existing and new event type are asked?

Reply:
If only the event type is required it also seems better to make this an argument.

Reply:
Agreed, I had flexibility in mind, but that might be overkill for this case.

Reply:
I would either go for full flexibility (asking the "from" and "to"), or none at all and then this command could just go over all known incorrect types and convert them to the right ones.
I don't think there's a use case to run the command for one incorrect type on its own.